azure/functions/blob.py

```diff
@@ -75,6 +75,9 @@ def encode(cls, obj: typing.Any, *,
 
     @classmethod
     def decode(cls, data: meta.Datum, *, trigger_metadata) -> typing.Any:
+        if data is None or data.type is None:
+            return None
+            
```

Comment:
Hi @kolafsson, thanks for contributing to azure-functions-python-library

Please add a few unit tests regarding to this scenario.
Unit test examples: https://github.com/Azure/azure-functions-python-library/pull/28/files#diff-c27162ae93fd2aa43a783ef13d1749f4R9-R12

Thanks.